docs/samples/sample.ts

```diff
@@ -3,6 +3,7 @@ import {createClient} from '@usevenice/openapi-client'
 import {createSdk, createVeniceClient} from '@usevenice/sdk'
 import discordSdkDef from '@usevenice/sdk-discord'
 import slackSdkDef from '@usevenice/sdk-slack'
+import openaiSdkDef from '../../opensdks/sdk-openai'
```

Comment:
Don't use path import from a diff package. Also follow pattern from above

Reply:
following that leads to 
```
Unable to resolve path to module '@usevenice/sdk-openai'.eslint[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.29.0/docs/rules/no-unresolved.md)
```